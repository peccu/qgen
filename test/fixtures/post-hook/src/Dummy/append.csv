Old,Lines
Old,Lines2
