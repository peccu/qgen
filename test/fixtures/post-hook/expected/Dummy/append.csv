Old,Lines
Old,Lines2
Dummy,dummy
